src/scss/_drawer.scss

```diff
@@ -0,0 +1,6 @@
+.registry__drawer.o-header__drawer {
+	.o-header__drawer-tools {
+		@include oTypographySansBold(1);
+		padding: 12px 0 4px 16px; //makes the drawer tool strip the same height as the header bar
```

Comment:
Is this something we should put in `o-header` at all?

Reply:
I think probably yes, but I was wary of updating it because of rewriting stuff? If the products that use this require it, then lets add it for sure.